integreat_cms/cms/templates/push_notifications/push_notification_template_list.html

```diff
@@ -0,0 +1,63 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load push_notification_filters %}
+{% block content %}
+    <div class="table-header">
+        <h1 class="heading">
+            {% translate "Templates" %}
+        </h1>
+        <div class="flex flex-wrap justify-end gap-4">
+            {% if perms.cms.change_pushnotification %}
+                <a href="{% url 'new_push_notification' region_slug=request.region.slug language_slug=language.slug %}"
+                   class="btn">
+                    {% translate "Create news message" %}
```

Comment:
Since we are in the template push notiifcation list view, I would expect that clicking this button creates a new template, not a 'normal' pn

Reply:
@david-venhoff would it be okay if we automatically set the checkbox?

Reply:
sure!

Reply:
Fixed.